src/D2L.CodeStyle.Analyzers/ApiUsage/DependencyInjection/Domain/ConfigureInstancePluginsExpression.cs

```diff
@@ -24,7 +24,12 @@ internal sealed class ConfigureInstancePluginsExpression : DependencyRegistratio
 				&& method.Parameters.Length == 1 );
 		}
 
-		internal override DependencyRegistration GetRegistration( IMethodSymbol method, SeparatedSyntaxList<ArgumentSyntax> arguments, SemanticModel semanticModel ) {
+		internal override DependencyRegistration GetRegistration(

```

Comment:
Series of plugins where only one of them needed the `CancellationToken `

Reply:
I'm hoping we can delete these ones soon because of static DI

Reply:
(but this change is good)